### PR TITLE
added the pet surrender admin form

### PR DESCRIPTION
**Pull request by Krabby-Patties**

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import PetTypesIndex from "./PetTypesIndex"
 import PetShowPage from "./PetShowPage"
 import ViewAdoptionForms from "./ViewAdoptionForms"
 import AdoptedPetsIndex from "./AdoptedPetsIndex"
-
+import ViewSurrenderForm from "./ViewSurrenderForm"
 const Navbar = props => {
```
Comment (DanAlberts) [inline src/components/Navbar.js:11]: You will want a blank line between lines 9 and 10

File: src/components/ViewSurrenderForm.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    useEffect(() => {
+        fetch("/api/v1/surrender_application").then((response) => response.json())
+            .then(surrenderForms => {
+                setAllSurrenderForms(surrenderForms.rows)
+            })}
+        , [])
+
+    let viewAllForms = allSurrenderForms.map(SurrenderForm => {
+    return <option key={SurrenderForm.id} value={JSON.stringify(SurrenderForm)}>--{`${SurrenderForm.name}, Surrender Application #${SurrenderForm.id}, ${SurrenderForm.application_status}`}--</option>
```
Comment (DanAlberts) [inline src/components/ViewSurrenderForm.js]: this return should be indented

File: src/components/ViewSurrenderForm.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            <label htmlFor="approval">Choose Approval Status:</label>
+            <select onChange={handleSurrenderChange} value={newSurrender} name="approval" id="approval">
+                <option value="">--Please choose an option--</option>
+                <option value="approved">Approve</option>
+                <option value="denied">Deny</option>
+            </select>
+
+            <input type="submit" value="Submit" />
+        </form>)
+
```
Comment (DanAlberts) [inline src/components/ViewSurrenderForm.js:84]: Unnecessary blank line here

**Review by DanAlberts — CHANGES_REQUESTED** [review]

File: src/components/ViewSurrenderForm.js (hunk 2 above)
Comment (Krabby-Patties) [reply src/components/ViewSurrenderForm.js]: fixed

**Review by DanAlberts — APPROVED** [review]

**Pull request merged**